platform-config/config/themes/field/sites.json

```diff
@@ -0,0 +1,19 @@
+{
+  "$id": "c_sites",
```

Comment:
I know "site" is a little overloaded so open to a better name for this field, which is the field that exists on a Pages Theme that links to the Site entity it is being used in